python/utils.py

```diff
@@ -30,21 +30,14 @@ class BuildConfig:
     version_tag: str = None
     build_type: str = "Release"
     wheel_name: str = "nvfuser"
-    nvmmh_include_dir: str = None
+    nvmmh_include_dir: str = ""
```

Comment:
How does this fix the problem? I saw `/opt/pytorch/nvfuser/python/None` so there seemed to be a problematic string join somewhere.